docs/CONFIGURATION.md

```diff
@@ -0,0 +1,270 @@
+## For English press [here](#configuration)
+# Konfiguracja
+
+Ten dokument zawiera przegląd wartości konfigurowalnych w pliku [`values.yaml`](../charts/kam/values.yaml) dla chartu Helm. Te wartości mogą być dostosowywane w celu skonfigurowania wdrożenia aplikacji. W celu uzyskania szczegółowych informacji jak wdrożyć aplikację, odwołaj się do pliku [DEPLOYMENT.md](./DEPLOYMENT.md).
+
+## Ogólna konfiguracja
+
+- **nameOverride**: Nadpisuje nazwę chartu. Przydatne, jeśli chcesz wdrożyć wiele instancji chartu z różnymi nazwami.
+- **fullnameOverride**: Całkowicie nadpisuje wygenerowaną nazwę chartu.
+- **podAnnotations**: Adnotacje do dodania do zasobów pod.
+- **podLabels**: Etykiety do dodania do zasobów pod.
+
+## Globalna konfiguracja
+
+### **global.env.FRONTEND_KEYCLOAK_URL**
+- **Opis**: URL do serwera Keycloak używanego przez frontend. Jest wykorzystywany przez frontend do celów uwierzytelniania. Powinien być publicznie dostępny.
+- **Wymagane**: Tak
+- **Domyślne**: Brak
+- **Używane przez**: Frontend
+- **Przykład**: `https://keycloak.example.com`
+
+### **global.env.KEYCLOAK_REALM_NAME**
+- **Opis**: Nazwa realm w Keycloak. Realm w Keycloak to przestrzeń, w której zarządzasz obiektami, takimi jak użytkownicy, role i klienci.
+- **Wymagane**: Tak
+- **Domyślne**: Brak
+- **Używane przez**: Frontend i Backend
+- **Przykład**: `myrealm`
+
+### **global.env.KEYCLOAK_CLIENT_NAME**
+- **Opis**: Nazwa klienta w Keycloak. Klient w Keycloak to podmiot, który może poprosić o uwierzytelnienie użytkownika. Należy utworzyć klienta w Keycloak dla naszej aplikacji.
+- **Wymagane**: Tak
```

Comment:
For backend mandatory, for frontend as wyżej xd

Reply:
I would argue it shouldn't be mandatory for backend, what if admin doesn't want to read in any client level roles for authorization, what then(I dont see it used anywhere else)? I'll add that change and any other leftovers in this PR comments to https://github.com/ZPI-2024-25/KubernetesAccessManager/pull/58